### ISSUE TITLE
Plans in both iOS and web showing previous clients after FTA client updates change

### ISSUE THREAD

**Issue by LisaMoore1** [bcgov/range-ios#158]
**Background**
- agreement holders change by way of a disposition
- when the disposition is complete staff update FTA to reflect the new agreement holders
- if a new plan is needed as a result of client changes staff can initiate an amendment so the need for signatories is triggered
- identified by Kassia James on  RAN076112 (2019-03-15)

**Requirements-General**
- [ ] only current agreement holders/partners should show up on current/editable plans
- [ ] former agreement holders must be identified and shown as having signed on the plans to which they were signatories
- [ ] once a plan is approved (signed by all) it should not longer "accept" FTA update changes to data
- [ ] when an approved plan enters (or remains in) any editable status (ex. by way of amendment) "FTA current" data needed

**Requirements-Specific**
- [x] current clients pulled into system using initial data pull from FTA, identified using Client# and locn code and visible on pertinent RANs 
- [ ] on subsequent data pulls current vs. previous and main vs. joint venture partner status must be updated
- [ ] agreement holders displayed consistent with Requirements-General
